cita-network/src/node_manager.rs

```diff
@@ -286,6 +286,8 @@ impl NodesManager {
         }
     }
 
+    // clippy
+    #[allow(clippy::drop_copy, clippy::zero_ptr)]
```

Comment:
Why not fix it?

Reply:
for clippy

Reply:
maybe you can try it first

Reply:
it is up to lib's macro select!

Reply:
it is due to use new crossbeam-channel and the reson for use crossbeam-channel,is upgrade cita-logger in bft

Reply:
ok